### PR TITLE
DOC-9671 PR #119205 - cli: --include-range-info flag for `cockroach debug zip` also toggles problem ranges

### PR DESCRIPTION
**Pull request by florence-crl**
Fixes DOC-9671

In cockroach-debug.zip, (a) removed problem ranges from “cannot be filtered list”, (b) updated to description of --include-range-info flag to include problemranges.json.

Rendered preview: [cockroach debug zip](https://deploy-preview-18512--cockroachdb-docs.netlify.app/docs/v24.1/cockroach-debug-zip.html)

### CODE REVIEW

**Review by nkodali — APPROVED** [review]

File: src/current/v23.1/cockroach-debug-zip.md [hunk 1]
```diff
@@ -114,7 +113,7 @@ Flag | Description
...
+`--include-range-info` | Include one file per node with information about the KV ranges stored on that node, in `nodes/{node ID}/ranges.json`.<br /><br />This information can be vital when debugging issues that involve the [KV layer]({% link {{ page.version.version }}/architecture/overview.md %}#layers) (which includes everything below the SQL layer), such as data placement, load balancing, performance or other behaviors. In certain situations, on large clusters with large numbers of ranges, these files can be omitted if and only if the issue being investigated is already known to be in another layer of the system (for example, an error message about an unsupported feature or incompatible value in a SQL schema change or statement). Note however many higher-level issues are ultimately related to the underlying KV layer described by these files so only set this to `false` if directed to do so by Cockroach Labs support.<br /><br />In addition, include problem ranges information in `reports/problemranges.json`.<br /><br />**Default:** true
```
Comment (taroface) [inline src/current/v23.1/cockroach-debug-zip.md:116]:
```suggestion
`--include-range-info` | Include one file per node with information about the KV ranges stored on that node, in `nodes/{node ID}/ranges.json`.<br /><br />This information can be vital when debugging issues that involve the [KV layer]({% link {{ page.version.version }}/architecture/overview.md %}#layers) (which includes everything below the SQL layer), such as data placement, load balancing, performance, or other behaviors. In certain situations, on large clusters with large numbers of ranges, these files can be omitted if and only if the issue being investigated is already known to be in another layer of the system (for example, an error message about an unsupported feature or incompatible value in a SQL schema change or statement). However, many higher-level issues are ultimately related to the underlying KV layer described by these files. Only set this to `false` if directed to do so by Cockroach Labs support.<br /><br />In addition, include problem ranges information in `reports/problemranges.json`.<br /><br />**Default:** true
```

File: src/current/v23.2/cockroach-debug-zip.md [hunk 2]
```diff
@@ -114,7 +113,7 @@ Flag | Description
...
+`--include-range-info` | Include one file per node with information about the KV ranges stored on that node, in `nodes/{node ID}/ranges.json`.<br /><br />This information can be vital when debugging issues that involve the [KV layer]({% link {{ page.version.version }}/architecture/overview.md %}#layers) (which includes everything below the SQL layer), such as data placement, load balancing, performance or other behaviors. In certain situations, on large clusters with large numbers of ranges, these files can be omitted if and only if the issue being investigated is already known to be in another layer of the system (for example, an error message about an unsupported feature or incompatible value in a SQL schema change or statement). Note however many higher-level issues are ultimately related to the underlying KV layer described by these files so only set this to `false` if directed to do so by Cockroach Labs support.<br /><br />In addition, include problem ranges information in `reports/problemranges.json`.<br /><br />**Default:** true
```
Comment (taroface) [inline src/current/v23.2/cockroach-debug-zip.md:116]: Ditto

File: src/current/v24.1/cockroach-debug-zip.md [hunk 3]
```diff
@@ -114,7 +113,7 @@ Flag | Description
...
+`--include-range-info` | Include one file per node with information about the KV ranges stored on that node, in `nodes/{node ID}/ranges.json`.<br /><br />This information can be vital when debugging issues that involve the [KV layer]({% link {{ page.version.version }}/architecture/overview.md %}#layers) (which includes everything below the SQL layer), such as data placement, load balancing, performance or other behaviors. In certain situations, on large clusters with large numbers of ranges, these files can be omitted if and only if the issue being investigated is already known to be in another layer of the system (for example, an error message about an unsupported feature or incompatible value in a SQL schema change or statement). Note however many higher-level issues are ultimately related to the underlying KV layer described by these files so only set this to `false` if directed to do so by Cockroach Labs support.<br /><br />In addition, include problem ranges information in `reports/problemranges.json`.<br /><br />**Default:** true
```
Comment (taroface) [inline src/current/v24.1/cockroach-debug-zip.md:116]: Ditto

**Review by taroface — APPROVED** [review]
LGTM, small nits

File: src/current/v23.1/cockroach-debug-zip.md (hunk 1 above)
Comment (florence-crl) [reply src/current/v23.1/cockroach-debug-zip.md:116]: updated

File: src/current/v23.2/cockroach-debug-zip.md (hunk 2 above)
Comment (florence-crl) [reply src/current/v23.2/cockroach-debug-zip.md:116]: updated

File: src/current/v24.1/cockroach-debug-zip.md (hunk 3 above)
Comment (florence-crl) [reply src/current/v24.1/cockroach-debug-zip.md:116]: updated

**Pull request merged**